ieeeuoft/makeuoft-site-2021
# Update the dashboard to not use RSVP

## Overview

- Resolves #13 
- <Description of all changes made and features implemented>


## Unit Tests Created

None. 

Removed the tests checking for different RSVP conditions.

Modified tests that were checking for the existence of RSVP buttons or lack thereof. The view no longer exists so the check is meaningless and will throw an error.

## Steps to QA

- Create a user and try accepting, waitlisting, and rejecting them. Check the dashboard prior and after sending the decision.
